### ISSUE TITLE
Composing styles with css using a prop

### ISSUE THREAD

**Issue by BrandonWade** [emotion-js/emotion#303]
- `emotion` version: 7.2.2
- `react` version: 15.6.1

Relevant code.

```javascript
// This works
let backdropStyle = animation.getBackdropStyle(closing);
if (this.props.backdropStyle) {
    backdropStyle = css`
        composes: ${backdropStyle};
        background-color: orange;
    `;
}

// This does not work
let backdropStyle = animation.getBackdropStyle(closing);
if (this.props.backdropStyle) {
    backdropStyle = css`
        composes: ${backdropStyle};
        ${this.props.backdropStyle} // Prop is a string with value background-color: orange;
    `;
}
```

What you did:
Attempted to compose styles with `css` by passing some styles in via a prop.

What happened:
Passing a prop in with a valid CSS string does not seem to work, yet hard-coding the value passed in directly works as expected.

Context:
I'm working on upgrading a package which has some default styles defined in JS to use emotion. Using `css` on those styles is working as expected. The user may optionally pass in a prop to override these styles - a string containing the CSS values to override. I'm attempting to use `composes` to overwrite the default styles with the values being passed in via a prop, but it doesn't seem to work. Hard-coding the values directly into the string tagged with `css` works however. Is this intended behaviour?

**Comment by mitchellhamilton** [comment]
This is expected behaviour in emotion 7, this is because in v7 we turn tagged template literals into regular function calls with styles as objects. Once #299 is done this will work but for now making the `backdropStyle` prop a style object will also work.

**Comment by BrandonWade** [comment]
Thanks, that did the trick! Are object styles still not auto-prefixed in v7? The article on medium says they aren't, but that looks like it predates v7.

**Comment by mitchellhamilton** [comment]
We try to prefix them but since in v7 we do it in the babel plugin it doesn't work in many cases such as this one. This will also be solved by #299.

**Comment by BrandonWade** [comment]
Okay. Thanks for the quick reply and your help!
